### ISSUE TITLE
failed to pip2.7 install korg

### ISSUE THREAD

**Issue by keevee09** [finklabs/korg#6]
Here is the output from >pip install korg
with homebrew python2.7 in a virtualenv:

(vpy2) tony@Macintosh:~/tmp/test_korg$ pip install korg
Collecting korg
  Using cached korg-0.0.5.tar.gz
Collecting regex>=2013-06-05 (from korg)
  Using cached regex-2016.09.22.tar.gz
Building wheels for collected packages: korg, regex
  Running setup.py bdist_wheel for korg ... error
  Complete output from command /Users/tony/virtualenvs/vpy2/bin/python2.7 -u -c "import setuptools, tokenize;__file__='/private/var/folders/cp/7b100dx88xl5tt006s6n17q80000gn/T/pip-build-DtfrTe/korg/setup.py';exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\r\n', '\n'), __file__, 'exec'))" bdist_wheel -d /var/folders/cp/7b100dx88xl5tt006s6n17q80000gn/T/tmpLCtjjxpip-wheel- --python-tag cp27:
  running bdist_wheel
  running build
  running build_py
  creating build
  creating build/lib
  creating build/lib/korg
  copying korg/__init__.py -> build/lib/korg
  copying korg/korg.py -> build/lib/korg
  copying korg/pattern.py -> build/lib/korg
  installing to build/bdist.macosx-10.11-x86_64/wheel
  running install
  running install_lib
  creating build/bdist.macosx-10.11-x86_64
  creating build/bdist.macosx-10.11-x86_64/wheel
  creating build/bdist.macosx-10.11-x86_64/wheel/korg
  copying build/lib/korg/__init__.py -> build/bdist.macosx-10.11-x86_64/wheel/korg
  copying build/lib/korg/korg.py -> build/bdist.macosx-10.11-x86_64/wheel/korg
  copying build/lib/korg/pattern.py -> build/bdist.macosx-10.11-x86_64/wheel/korg
  running install_data
  creating build/bdist.macosx-10.11-x86_64/wheel/korg-0.0.5.data
  creating build/bdist.macosx-10.11-x86_64/wheel/korg-0.0.5.data/data
  creating build/bdist.macosx-10.11-x86_64/wheel/korg-0.0.5.data/data/patterns
  copying patterns/aws -> build/bdist.macosx-10.11-x86_64/wheel/korg-0.0.5.data/data/patterns
  copying patterns/bacula -> build/bdist.macosx-10.11-x86_64/wheel/korg-0.0.5.data/data/patterns
  copying patterns/bro -> build/bdist.macosx-10.11-x86_64/wheel/korg-0.0.5.data/data/patterns
  copying patterns/exim -> build/bdist.macosx-10.11-x86_64/wheel/korg-0.0.5.data/data/patterns
  copying patterns/firewalls -> build/bdist.macosx-10.11-x86_64/wheel/korg-0.0.5.data/data/patterns
  copying patterns/grok-patterns -> build/bdist.macosx-10.11-x86_64/wheel/korg-0.0.5.data/data/patterns
  copying patterns/haproxy -> build/bdist.macosx-10.11-x86_64/wheel/korg-0.0.5.data/data/patterns
  copying patterns/java -> build/bdist.macosx-10.11-x86_64/wheel/korg-0.0.5.data/data/patterns
  copying patterns/junos -> build/bdist.macosx-10.11-x86_64/wheel/korg-0.0.5.data/data/patterns
  copying patterns/linux-syslog -> build/bdist.macosx-10.11-x86_64/wheel/korg-0.0.5.data/data/patterns
  copying patterns/mcollective -> build/bdist.macosx-10.11-x86_64/wheel/korg-0.0.5.data/data/patterns
  copying patterns/mcollective-patterns -> build/bdist.macosx-10.11-x86_64/wheel/korg-0.0.5.data/data/patterns
  copying patterns/mongodb -> build/bdist.macosx-10.11-x86_64/wheel/korg-0.0.5.data/data/patterns
  copying patterns/nagios -> build/bdist.macosx-10.11-x86_64/wheel/korg-0.0.5.data/data/patterns
  copying patterns/postgresql -> build/bdist.macosx-10.11-x86_64/wheel/korg-0.0.5.data/data/patterns
  copying patterns/rails -> build/bdist.macosx-10.11-x86_64/wheel/korg-0.0.5.data/data/patterns
  copying patterns/redis -> build/bdist.macosx-10.11-x86_64/wheel/korg-0.0.5.data/data/patterns
  copying patterns/ruby -> build/bdist.macosx-10.11-x86_64/wheel/korg-0.0.5.data/data/patterns
  warning: install_data: setup script did not provide a directory for 'README.md' -- installing right in 'build/bdist.macosx-10.11-x86_64/wheel/korg-0.0.5.data/data'

  error: can't copy 'README.md': doesn't exist or not a regular file

  ----------------------------------------
  Failed building wheel for korg
  Running setup.py clean for korg
  Running setup.py bdist_wheel for regex ... done
  Stored in directory: /Users/tony/Library/Caches/pip/wheels/58/15/f6/ee4b932c377334b771607ea57b104b801904042c6715d60532
Successfully built regex
Failed to build korg
Installing collected packages: regex, korg
  Running setup.py install for korg ... error
    Complete output from command /Users/tony/virtualenvs/vpy2/bin/python2.7 -u -c "import setuptools, tokenize;__file__='/private/var/folders/cp/7b100dx88xl5tt006s6n17q80000gn/T/pip-build-DtfrTe/korg/setup.py';exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\r\n', '\n'), __file__, 'exec'))" install --record /var/folders/cp/7b100dx88xl5tt006s6n17q80000gn/T/pip-WuvVg_-record/install-record.txt --single-version-externally-managed --compile --install-headers /Users/tony/virtualenvs/vpy2/bin/../include/site/python2.7/korg:
    running install
    running build
    running build_py
    creating build
    creating build/lib
    creating build/lib/korg
    copying korg/__init__.py -> build/lib/korg
    copying korg/korg.py -> build/lib/korg
    copying korg/pattern.py -> build/lib/korg
    running install_lib
    creating /Users/tony/virtualenvs/vpy2/lib/python2.7/site-packages/korg
    copying build/lib/korg/__init__.py -> /Users/tony/virtualenvs/vpy2/lib/python2.7/site-packages/korg
    copying build/lib/korg/korg.py -> /Users/tony/virtualenvs/vpy2/lib/python2.7/site-packages/korg
    copying build/lib/korg/pattern.py -> /Users/tony/virtualenvs/vpy2/lib/python2.7/site-packages/korg
    byte-compiling /Users/tony/virtualenvs/vpy2/lib/python2.7/site-packages/korg/__init__.py to __init__.pyc
    byte-compiling /Users/tony/virtualenvs/vpy2/lib/python2.7/site-packages/korg/korg.py to korg.pyc
    byte-compiling /Users/tony/virtualenvs/vpy2/lib/python2.7/site-packages/korg/pattern.py to pattern.pyc
    running install_data
    creating /Users/tony/virtualenvs/vpy2/patterns
    copying patterns/aws -> /Users/tony/virtualenvs/vpy2/patterns
    copying patterns/bacula -> /Users/tony/virtualenvs/vpy2/patterns
    copying patterns/bro -> /Users/tony/virtualenvs/vpy2/patterns
    copying patterns/exim -> /Users/tony/virtualenvs/vpy2/patterns
    copying patterns/firewalls -> /Users/tony/virtualenvs/vpy2/patterns
    copying patterns/grok-patterns -> /Users/tony/virtualenvs/vpy2/patterns
    copying patterns/haproxy -> /Users/tony/virtualenvs/vpy2/patterns
    copying patterns/java -> /Users/tony/virtualenvs/vpy2/patterns
    copying patterns/junos -> /Users/tony/virtualenvs/vpy2/patterns
    copying patterns/linux-syslog -> /Users/tony/virtualenvs/vpy2/patterns
    copying patterns/mcollective -> /Users/tony/virtualenvs/vpy2/patterns
    copying patterns/mcollective-patterns -> /Users/tony/virtualenvs/vpy2/patterns
    copying patterns/mongodb -> /Users/tony/virtualenvs/vpy2/patterns
    copying patterns/nagios -> /Users/tony/virtualenvs/vpy2/patterns
    copying patterns/postgresql -> /Users/tony/virtualenvs/vpy2/patterns
    copying patterns/rails -> /Users/tony/virtualenvs/vpy2/patterns
    copying patterns/redis -> /Users/tony/virtualenvs/vpy2/patterns
    copying patterns/ruby -> /Users/tony/virtualenvs/vpy2/patterns
    warning: install_data: setup script did not provide a directory for 'README.md' -- installing right in '/Users/tony/virtualenvs/vpy2'

    error: can't copy 'README.md': doesn't exist or not a regular file

    ----------------------------------------
Command "/Users/tony/virtualenvs/vpy2/bin/python2.7 -u -c "import setuptools, tokenize;__file__='/private/var/folders/cp/7b100dx88xl5tt006s6n17q80000gn/T/pip-build-DtfrTe/korg/setup.py';exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\r\n', '\n'), __file__, 'exec'))" install --record /var/folders/cp/7b100dx88xl5tt006s6n17q80000gn/T/pip-WuvVg_-record/install-record.txt --single-version-externally-managed --compile --install-headers /Users/tony/virtualenvs/vpy2/bin/../include/site/python2.7/korg" failed with error code 1 in /private/var/folders/cp/7b100dx88xl5tt006s6n17q80000gn/T/pip-build-DtfrTe/korg/

**Comment by markfink** [comment]
thanks for the update - should be fixed with v0.0.6. let me know if it works on Mac, too
```bash
(vkorg) mark@ichikawa:~/tmp$ pip install korg
Collecting korg
  Downloading korg-0.0.6.tar.gz
Collecting regex>=2013-06-05 (from korg)
Building wheels for collected packages: korg
  Running setup.py bdist_wheel for korg ... done
  Stored in directory: /home/mark/.cache/pip/wheels/99/dd/34/b78763f28d62aaa60d3b05cf19c291df283c1b4ab75cc47ca8
Successfully built korg
Installing collected packages: regex, korg
Successfully installed korg-0.0.6 regex-2016.9.22
```

**Comment by keevee09** [comment]
it's working on Mac again also
```
(v2) tony@Macintosh:~/tmp$ pip install korg
Collecting korg
  Downloading korg-0.0.6.tar.gz
Collecting regex>=2013-06-05 (from korg)
  Using cached regex-2016.09.22.tar.gz
Building wheels for collected packages: korg, regex
  Running setup.py bdist_wheel for korg ... done
  Stored in directory: /Users/tony/Library/Caches/pip/wheels/99/dd/34/b78763f28d62aaa60d3b05cf19c291df283c1b4ab75cc47ca8
  Running setup.py bdist_wheel for regex ... done
  Stored in directory: /Users/tony/Library/Caches/pip/wheels/58/15/f6/ee4b932c377334b771607ea57b104b801904042c6715d60532
Successfully built korg regex
Installing collected packages: regex, korg
Successfully installed korg-0.0.6 regex-2016.9.22
```
